charts/bulk-scan-processor/values.yaml

```diff
@@ -1,15 +1,35 @@
 java:
   applicationPort: 8581
+  ingressHost: bulk-scan-processor-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   environment:
     LOGBACK_REQUIRE_ALERT_LEVEL: "false"
     LOGBACK_REQUIRE_ERROR_CODE: "false"
     S2S_NAME: "bulk_scan_processor"
     SMTP_HOST: "false"
     S2S_URL: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
     DOCUMENT_MANAGEMENT_URL: "http://dm-store-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    SCAN_ENABLED: "true"
+    SCAN_DELAY: "4000"
+    REUPLOAD_ENABLED: "true"
+    REUPLOAD_DELAY: "4000"
+    SPRING_MAIL_HOST: "false"
+    NOTIFICATIONS_TO_ORCHESTRATOR_TASK_ENABLED: "true"
+    NOTIFICATIONS_TO_ORCHESTRATOR_TASK_DELAY: "3000"
+    STORAGE_BLOB_SELECTED_CONTAINER: "ALL"
+    STORAGE_BLOB_SIGNATURE_ALGORITHM: "sha256withrsa"
+    STORAGE_BLOB_PUBLIC_KEY: "nonprod_public_key.der"
+    BULK_SCANNING_DB_USER_NAME: "{{ .Values.postgresql.postgresqlUsername}}"
+    BULK_SCANNING_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+    BULK_SCANNING_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
+    BULK_SCANNING_DB_HOST: "{{ .Release.Name }}-postgresql"
+    BULK_SCANNING_DB_PORT: "5432"
+    FLYWAY_URL: "jdbc:postgresql://{{ .Release.Name }}-postgresql:5432/{{ .Values.postgresql.postgresqlDatabase}}"
+    FLYWAY_USER: "{{ .Values.postgresql.postgresqlUsername}}"
+    FLYWAY_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+    FLYWAY_NOOP_STRATEGY: "false"
   keyVaults:
-    s2s:
-      resourceGroup: rpe-service-auth-provider
+    "bulk-scan":
```

Comment:
so basically we don't use multiple key-vaults and just grab everything from product vault e.g.

```yaml
s2s-secret: S2S_SECRET
```

down below